tests/pyccel/test_pyccel.py

```diff
@@ -137,6 +137,8 @@ def compare_pyth_fort_output_by_type( p_output, f_output, dtype=float, language=
     if dtype is str:
         p_list = [e.strip() for e in re.split('\n', p_output)]
         f_list = [e.strip() for e in re.split('\n', f_output)]
+        print(p_list)
+        print(f_list)
```

Comment:
remove debugging prints

Reply:
Good catch! Please can you have a look at the comments that you left in your last review and decide if those conversations can be closed too.

Reply:
Fixed 2b2dfb17

Reply:
> Good catch! Please can you have a look at the comments that you left in your last review and decide if those conversations can be closed too.

Ok, sure I will